tests/contrib/rq/test_rq.py

```diff
@@ -134,7 +135,10 @@ def test_enqueue(queue, distributed_tracing_enabled, worker_service_name):
         distributed_tracing_enabled,
         worker_service_name,
     )
-    with snapshot_context(token, ignores=snapshot_ignores):
+    num_traces_expected = None
+    if sys.version_info.major == 3 and sys.version_info.minor != 5:
```

Comment:
do we need this for Python 2.7?

should we have something like this instead?

```suggestion
    if not (sys.version_info.major == 3 and sys.version_info.minor == 5):
```

Reply:
Thanks for the better readability suggestion. This works fine with Python 2.7.

Reply:
weird that it works with 2.7, since we'd never enter this branch, so `num_traces_expected` would always be `None` ?

Reply:
Ahh, I didn't realize this we'd never enter the branch in Python 2.7. Now we do and it passes for sure 😄 
In hindsight it's not too strange that it passed with Python 2.7, since this was only a mildly flaky test and still passes for the most part.

Reply:
yeah, exactly!